.husky/pre-commit

```diff
@@ -6,6 +6,14 @@ if [ "$branch" = "main" ]; then
   exit 1
 fi
 
+# Revert updates to storybookLoader.js autogenerated by `storybook` in the React Native Example app
+story_loader_path="examples/react-native/storybook/storyLoader.js"
+status="$(git status --porcelain)"
+if [[ $status == *$story_loader_path* ]]; then
+  git checkout HEAD -- $story_loader_path
+  echo "Revert updates to $story_loader_path"
+fi
+
```

Comment:
This feels a little over-engineered, why not just add the `rnstl` to the dev script? it's not a time-consuming operation.

```
"dev": "rnstl && react-native start"
```

Also because you need to run storybook server in order to generate this file, so if you wanted for instance to change the env var and enable storybook, but just run the app without the web server, then you wouldn't see the stories, or risk not seeing updated stories.

Reply:
Valid concern but this approach is due to the fact that this App is not just for `storybook` usage and we need to isolate `storybook` outside of our e2e tests

Reply:
I added `react-native-storybook-loader` as a convenience tool, so we don't have to manually maintain a file with imports for all our stories. But if we need to add workarounds it's becoming inconvenient and maybe the right thing to do is to take a step back and remove this util. We can always add it back later, and since all our stories will be in one folder, adding the imports is not too big of a hassle for now. What do you think?